hyperspy/misc/io/fei_stream_readers.py

```diff
@@ -202,7 +203,9 @@ def stream_to_sparse_COO_array(
             first_frame=first_frame,
             last_frame=last_frame,
         )
-    return DenseSliceCOO(coords=coords, data=data, shape=shape)
+    dense_sparse = DenseSliceCOO(coords=coords, data=data, shape=shape)
+    dask_sparse = da.from_array(dense_sparse, chunks="auto")
```

Comment:
Following #1995, would it be sensible to start passing `chunks` parameters from the load function down to reader, in this case, here?

Reply:
Yes, but I suggest to leave it for another PR.